edx_exams/apps/api/v1/views.py

```diff
@@ -212,26 +212,34 @@ def handle_config(cls, provider, course_id):
         CourseExamConfiguration.objects.update_or_create(
             course_id=course_id,
             defaults={'provider': provider})
-        log.info(f"Created or updated course exam configuration course_id={course_id},provider={provider.name}")
+        provider_name = provider.name if provider else None
+        log.info(f"Created or updated course exam configuration course_id={course_id},provider={provider_name}")
 
     def patch(self, request, course_id):
         """
         Create/update course exam configuration.
         """
+        error = None
+
         # check that proctoring provider is in request
-        if request.data.get('provider') is None:
-            response_status = status.HTTP_400_BAD_REQUEST
-            data = {"detail": "No proctoring provider name in request."}
+        if 'provider' not in request.data:
+            error = {"detail": "No proctoring provider name in request."}
+        elif request.data.get('provider') is None:
```

Comment:
Question: could you clarify the difference between the if and elif conditions? This basically allows for the provider to be 'None' in the case described in the PR description if the provider needs to be switched back from LTI to proctoring?

Reply:
this ensures you have to explicitly set the provider to `null` in the request. So the first check makes sure 'provider' is afield in the request and the second checks if the value itself is None/null.

Actually this reminds me I need to throw a test in here 💡